templates/dashboard/order/fulfillment.html

```diff
@@ -77,7 +77,7 @@ <h5>
                 <tr>
                   <td>
                     {% if line.variant %}
-                      <a href="{% url 'dashboard:variant-detail' pk=line.variant.pk %}">{{ line.product_name }}</a>
+                      <a href="{% url 'dashboard:variant-details' pk=line.variant.pk %}">{{ line.product_name }}</a>
```

Comment:
This is still incorrect, this URL takes `product_pk` and `variant_pk` as arguments.